Testscripts/Linux/nested_kvm_basic_test.sh

```diff
@@ -43,26 +52,28 @@ done
 ImageName="nested.qcow2"
 
 if [ -z "$NestedImageUrl" ]; then
-	echo "Please mention -NestedImageUrl next"
-	exit 1
+	LogErr "Please mention -NestedImageUrl next"
+	SetTestStateAborted
+	exit 0
 fi
 if [ -z "$HostFwdPort" ]; then
-	echo "Please mention -HostFwdPort next"
-	exit 1
+	LogErr "Please mention -HostFwdPort next"
+	SetTestStateAborted
+	exit 0
 fi
 if [ -z "$NestedUser" ]; then
-	echo "Please mention -NestedUser next"
-	exit 1
+	LogErr "Please mention -NestedUser next"
+	exit 0
```

Comment:
If the user and password are missing, it should be aborted also. Please add SetTestStateAborted.

Reply:
Updated. Thanks!